.github/workflows/scripts/validate-coderabbit.sh

```
@@ -0,0 +1,51 @@
+echo "Fetching PR reviews from GitHub API..."
+rate_limit=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+  "https://api.github.com/rate_limit")
+remaining=$(echo "$rate_limit" | jq '.rate.remaining')
+
+if [ "$remaining" -lt 1 ]; then
+  echo "Error: GitHub API rate limit exceeded. Please try again later."
+  exit 1
+fi
+
+reviews=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+  "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews")
```

Shellcheck (0.10.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)